index.js

```diff
@@ -289,8 +290,8 @@ function choosePluralForm(text, locale, count) {
 // interpolation placeholder.
 var dollarRegex = /\$/g;
 var dollarBillsYall = '$$';
-var tokenRegex = /%\{(.*?)\}/g;
-function interpolate(phrase, options) {
+function interpolate(phrase, options, tokenPattern) {
+  var tokenRegex = new RegExp(tokenPattern, 'g');
```

Comment:
no, i mean parse the token pattern. Allowing an API that makes it easy for developers to trigger an exception - especially when that exception would happen at interpolation time and not at instantiation time - is problematic.

At the very least, the new regex should be created in the constructor, to make the exception happen sooner.